src/pages/Chat/state.tsx

```diff
@@ -1,33 +1,68 @@
-import { Container, ContainerProps, StateUpdater } from 'constate'
+import { Container, ContainerProps, EffectProps } from 'constate'
+import { propOr } from 'ramda'
 import * as React from 'react'
+import { StorageContainer } from '../../utils/StorageContainer'
+
+export interface Step1State {
```

Comment:
I think we should name the steps with descriptive names - having them referred to by index is likely to get painful down the line.

Reply:
true!